File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/behavior/NavBarFormBehavior.java
package net.dontdrinkandroot.wicket.bootstrap.behavior;

import org.apache.wicket.Component;
import org.apache.wicket.behavior.Behavior;

import net.dontdrinkandroot.wicket.behavior.CssClassAppender;
import net.dontdrinkandroot.wicket.bootstrap.css.BootstrapCssClass;


public class NavBarFormBehavior extends Behavior
{

	@Override
	public void bind(Component component)
	{
		super.bind(component);
		component.add(new CssClassAppender(BootstrapCssClass.NAVBAR_FORM));
	}

}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/behavior/NavbarFormBehavior.kt
package net.dontdrinkandroot.wicket.bootstrap.behavior

import net.dontdrinkandroot.wicket.behavior.CssClassAppender
import net.dontdrinkandroot.wicket.bootstrap.css.BootstrapCssClass
import org.apache.wicket.Component
import org.apache.wicket.behavior.Behavior

class NavbarFormBehavior : Behavior() {

    override fun bind(component: Component) {
        super.bind(component)
        component.add(CssClassAppender(BootstrapCssClass.NAVBAR_FORM))
    }
}